owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewAudioFragment.kt

```diff
@@ -285,11 +301,13 @@ class PreviewAudioFragment : FileFragment() {
 
             R.id.action_set_available_offline -> {
                 fileOperationsViewModel.performOperation(FileOperation.SetFilesAsAvailableOffline(listOf(file)))
+                    Snackbar.make(requireView(), R.string.confirmation_set_available_offline, Snackbar.LENGTH_LONG).show()
```

Comment:
This line has an additional indentation. KtLint won't like that 😅

Reply:
Removed 🚀